### DISCUSSION TITLE
Issues with Windows & GPU

### DISCUSSION THREAD

**Discussion by JTMarsh556** [ggml-org/llama.cpp#6627] [Q&A]
I am having issues with getting GPU support with this.  
I tried to follow the instructions here but honestly the instructions were easier to follow on the langchain docs.  Can someone point me to something they know works?

Thanks

**Comment by Jeximo** [comment]
> I am having issues with getting GPU support with this.

It's unclear what the issue is exactly. *Did you use `main`, or access `llama.cpp` some other way?*

Here's the readme for how to enable GPU for `main`:
https://github.com/ggerganov/llama.cpp/blob/4e9a7f7f7fb6acbddd1462909c8d696e38edbfcc/examples/main/README.md?plain=1#L318

For example,
```
./main -m ~/model.gguf -ngl 99 -p "Building a website in 5 steps:"
```
